symphony/lib/core/class.session.php

```diff
@@ -183,7 +183,7 @@ public static function write($id, $data)
         // in the database. This prevents empty Sessions from being created, but
         // allows them to be nulled.
         $session_data = Session::read($id);
-        if (is_null($session_data)) {
+        if ($session_data === '') {
```

Comment:
Why not do `!$session_data` ? I would cover both cases ?

Reply:
Just didn't think of it :) Based on the current implementation, `Session::read` can't return anything _but_ a string.

Reply:
Yes but it would prevent future breakage ;) I really hate having `''` in my code, maybe I am crazy.

Reply:
Heh, ok, updated 👍